code/__DEFINES/dcs/signals.dm

```diff
@@ -440,3 +440,6 @@
 #define COMSIG_XENO_TURF_CLICK_SHIFT "xeno_turf_click_shift"				//from turf ShiftClickOn(): (/mob)
 #define COMSIG_XENO_TURF_CLICK_CTRL "xeno_turf_click_alt"					//from turf AltClickOn(): (/mob)
 #define COMSIG_XENO_MONKEY_CLICK_CTRL "xeno_monkey_click_ctrl"				//from monkey CtrlClickOn(): (/mob)
+
+// Religion Signals
+#define COMSIG_RELIGION_NEWSECT "rel_newsect"	//Post GLOB.religious_sect being set ()
```

Comment:
comeon indent and format it like the rest ;^)

Reply:
440-442 arent even the same smh